aiohttp/pytest_plugin.py

```diff
@@ -96,8 +96,14 @@ def finalizer():
 
 @pytest.fixture
 def fast(request):
```

Comment:
i would strongly recommend prefixing those fixtures as they have a overly generic name

Reply:
Agree.
The fixture exists for long time.
Bugfix release is a bad place for such changes.
But PR for master is welcome!

Reply:
im currently not using aiohttp myself so i'll have to defer